src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -38,7 +42,11 @@ type changeCache struct {
 	channelCaches   map[string]*channelCache // A cache of changes for each channel
 	onChange        func(base.Set)           // Client callback that notifies of channel changes
 	stopped         bool                     // Set by the Stop method
+	skippedSeqs     SkippedSequenceQueue     // Skipped sequences still pending on the TAP feed
+	skippedSeqLock  sync.RWMutex             // Coordinates access to skippedSeqs queue
 	lock            sync.RWMutex             // Coordinates access to struct fields
+	LateSeqLock     sync.RWMutex             // Coordinates access to late sequence caches
```

Comment:
should this be `lateSeqLock` rather than `LateSeqLock`?  It seems like it won't be exported anyway since the whole struct is unexported.